libs/ngx-charts-on-fhir/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@elimuinformatics/ngx-charts-on-fhir",
-  "version": "3.4.7",
+  "version": "3.5.0",
```

Comment:
This should be a major version upgrade (4.0.0) because it is not backwards-compatible with Angular 16